examples/counter/shared/src/app.rs

```diff
@@ -27,6 +28,7 @@ pub enum Event {
     Increment,
     Decrement,
     StartWatch,
+    SendUuid(Uuid),
```

Comment:
This is included as a sort of integration test, to verify that typegen works for Uuid in an example app. We can remove this before merging.